README.md

```diff
@@ -68,19 +85,22 @@ helm install <YOUR HELM RELEASE NAME> camunda/camunda-platform --set zeebe.prome
 
 You can remove these charts by running:
 
-```
+```sh
 helm uninstall <YOUR HELM RELEASE NAME>
 ```
+> **Note**
+>
+> Notice that all the services and pods will be deleted, but not the Persistence Volume Claims which are used to hold the storage for the data generated by the cluster and Elasticsearch.
 
-> Notice that all the services and pods will be deleted, but not the Persistence Volume Claims which are used to hold the storage for the data generated by the cluster and Elasticsearch. In order to free up the storage you need to manually delete all the Persistent Volume Claims. You can do this by running:
+In order to free up the storage you need to manually delete all the Persistent Volume Claims. You can do this by running:
 
-```
+```sh
```

Comment:
:wrench: We could also help the user with mentioning here the labels, like we do in our makefile

Reply:
sorry, I didn't get this one :sweat_smile:

which labels do you refer to?

Reply:
Sorry I meant something like this https://github.com/camunda/camunda-platform-helm/blob/main/Makefile#L59

Reply:
aha, I got it now :ok_hand: 

I've added it, also took a note about it because I believe that we need a single label for all resources even that installed by other packages (i.e. we need to label the es pvc resources so no need for the es label).

Reply:
agree

Reply:
Tbh If they would follow best practices we would need to specify it :sweat_smile: